lua/lvim/bootstrap.lua

```diff
@@ -112,11 +112,15 @@ end
 ---Update LunarVim
 ---pulls the latest changes from github and, resets the startup cache
 function M:update()
-  reload("lvim.utils.hooks").run_pre_update()
-  local ret = reload("lvim.utils.git").update_base_lvim()
-  if ret then
-    reload("lvim.utils.hooks").run_post_update()
-  end
+  require("lvim.core.log"):info "Trying to update LunarVim..."
+
+  vim.schedule(function()
```

Comment:
Is this still required?

Reply:
Yes, messages don't show up until the command exits otherwise (even vim.notify)